examples/threathunter/threat_intelligence/results.py

```diff
@@ -58,9 +58,13 @@ def normalize(self):
 
         if self.severity <= 0 or self.severity > 10:
             logging.warning("normalizing OOB score: {}".format(self.severity))
-            self.severity = max(1, min(self.severity, 10))
-            # NOTE: min 1 and not 0
-            # else err 400 from cbapi: Report severity must be between 1 & 10
+            if self.severity > 10 and self.severity < 100:
+                #assume it's a percentage
+                self.severity = round(self.severity)
```

Comment:
Are you assuming this will be a double `x.xx`?

Reply:
Oh this should be `round(self.severity/10)`
This assumes the number between 10 and 100, divided by 10, is a float